### PR TITLE
Add support jav321

### PR DESCRIPTION
**Pull request by 68cdrBxM8YdoJ**

### CODE REVIEW

File: jav321.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import requests
+from bs4 import BeautifulSoup
+from lxml import html
+
+
+def main(number: str) -> dict:
+    result = requests.post("https://www.jav321.com/search", data={"sn": number})
```
Comment (jnozsc) [inline jav321.py]: this change does not support the proxy from `config.ini `

you may want to take a look at https://github.com/yoshiko2/AV_Data_Capture/blob/master/ADC_function.py#L100  and do something similar

Comment (68cdrBxM8YdoJ) [reply jav321.py]: forgot about proxy...

**Comment by 68cdrBxM8YdoJ** [comment]
@jnozsc 
Add support proxy settings.
Please take a look.

**Comment by yoshiko2** [comment]
image
@68cdrBxM8YdoJ

**Comment by 68cdrBxM8YdoJ** [comment]
It worked properly on my pc.
I will look at it tomorrow.

**Comment by yoshiko2** [comment]
@jnozsc Do you have email address ? I want to Invite you to join the AVDC Developers group

**Pull request merged**